kratos/geometries/line_2d_2.h

```diff
@@ -792,6 +792,31 @@ class Line2D2 : public Geometry<TPointType>
     }
 
     /**
+     * It computes the normal of the geometry, if possible
+     * @return The normal of the geometry
+     */
+    virtual array_1d<double, 3> Normal() override
```

Comment:
Hi @loumalouomega, the point is that the API must be the same for all the geometries. This is a good reason for which a signature like

      virtual array_1d<double, 3> Normal()

should not be accepted: it assumes only one normal per element